src/handlers/app/types.ts

```diff
@@ -0,0 +1,5 @@
+export enum AppHandler {
+  CLOSE = 'app:close',
+  CHANGE_ROUTE = 'app:change-route',
+  SPLASHSCREEN_CLOSE = 'splashscreen:close',
```

Comment:
With the enum in place, it's probably less important now, but shouldn't this be `app:splashscreen-close`?

Reply:
That makes sense, I changed to `app:splashscreen-close` now!